test/hypercerts/IHypercertToken.sol

```diff
@@ -22,9 +22,7 @@ interface IHypercertToken {
     event ClaimStored(uint256 indexed claimID, string uri, uint256 totalUnits);
```

Comment:
remove the whole folder

Reply:
You sure there is nothing useful in that folder?

Reply:
yes, it's only there for the tests